### PR TITLE
Bunch of misc issues found using PVS-Studio

### PR DESCRIPTION
**Pull request by DarkLotus**
Some random errors, a few logic ones im not sure of ie Stealing, if someone wants to review/edit.

### CODE REVIEW

File: Scripts/Services/FireCasino/DiceGames.cs [hunk 1]
```diff
@@ -126,12 +126,6 @@ public virtual void Reset()
             BettingOn = -1;
             Stage = GameStage.Betting;
 
-            if (Roll != null)
```
Comment (kevin-10) [inline Scripts/Services/FireCasino/DiceGames.cs]: Is this going to cause an issue?

Comment (Vorspire) [reply Scripts/Services/FireCasino/DiceGames.cs]: It's fine, Roll is always nullified before being checked for not-null, meaning the if-statement was not necessary.

File: Scripts/Services/Monster Stealing/Core/StealingHandler.cs [hunk 2]
```diff
@@ -166,7 +166,7 @@ private static int GetStealingChance(Mobile thief, BaseCreature from, double ste
 
             if (level >= 40)
                 chance += 5;
-            else if (level >= 40)
+             else if (level >= 40) /*??*/
```
Comment (Vorspire) [inline Scripts/Services/Monster Stealing/Core/StealingHandler.cs]: Assume 40, 35, 30.

File: Scripts/Services/Expansions/Time Of Legends/Mobiles/DragonTurtle.cs [hunk 3]
```diff
@@ -174,7 +174,7 @@ public void DoBubble()
                             if (d == Direction.Up)
                                 d = Direction.North;
                             else
-                                d = d++;
+                                d = ++d;
```
Comment (Vorspire) [inline Scripts/Services/Expansions/Time Of Legends/Mobiles/DragonTurtle.cs]: Neither the old nor the new lines make sense here, left-hand assignment isn't necessary.

File: Scripts/Mobiles/NPCs/MysteriousWisp.cs [hunk 4]
```diff
@@ -497,7 +497,7 @@ public override void OnResponse(NetState state, RelayInfo info)
                 {
                     Item item = m_Items[idx];
 
-                    if (m_Wisp.Backpack.Items.Contains(item) && !item.Deleted && item != null)
+                    if (item != null && m_Wisp.Backpack.Items.Contains(item) && !item.Deleted)
```
Comment (Vorspire) [inline Scripts/Mobiles/NPCs/MysteriousWisp.cs]: Wisp Backpack should be null checked too.

File: Scripts/Services/FireCasino/DiceGames.cs (hunk 1 above)
Comment (kevin-10) [reply Scripts/Services/FireCasino/DiceGames.cs]: Haha, I should know, I wrote that!  It's been a while though.  Good catch.

Comment (DarkLotus) [reply Scripts/Services/FireCasino/DiceGames.cs]: Once i realized how many of these unneeded null checks there were(100+) i stopped removing them, none are in hot paths so shouldn't be a performance issue leaving them.

File: Scripts/Services/Monster Stealing/Core/StealingHandler.cs (hunk 2 above)
Comment (DarkLotus) [reply Scripts/Services/Monster Stealing/Core/StealingHandler.cs]: Will fix, i did browse uoguide and couldn't find anything related to fame for the formula so i was unsure of this one.

**Pull request merged**